tools/encodings/GF Glyph Sets/Cyrillic/filter lists/uni names/plus_unique-glyphs.txt

```diff
@@ -189,4 +191,4 @@ tugrik
 descender-cy
```

Comment:
Let's move these component-parts from the official list to recommended glyphs. These glyphs are used for component building and are non-exportable. I have seen people forget to set the non-export tag.

```
descender-cy  
descender-cy.case  
verticalbar-cy
verticalbar-cy.case
```